src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -241,4 +249,74 @@ describe("Payment", () => {
 
     expect(testProps.router.push).toHaveBeenCalledWith("/order2/1234/review")
   })
+
+  it("shows an error modal when there is an error in CreateCreditCardPayload", () => {
+    stripeMock.createToken.mockReturnValue({
+      then: func => func({ token: { id: "tokenId" } }),
+    })
+
+    mutationMock.mockImplementationOnce((_, { onCompleted }) =>
+      onCompleted(creatingCreditCardFailed)
+    )
+
+    const component = mount(<PaymentRoute {...testProps} />)
```

Comment:
#superminor and others may have the opposite opinion, but it feels like there are too many line breaks in these tests

Reply:
I just simply deleted the duplicate expectations since we don't have to test it in every test. We still have one test that checks this, but we may want to add a test dedicated for the modal behaviour rather than testing the integration with Modal/GraphQL.